pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -7,6 +7,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/jaegertracing/jaeger-operator/pkg/storage"
```

Comment:
Was it placed here by `make format`?

Reply:
yes 

It either places it after core imports or after 3rd party imports in a separate block. However when I put it into operator imports it leaves it there.